packages/components/tests/integration/components/hds/form/checkbox/field-test.js

```diff
@@ -89,10 +89,10 @@ module('Integration | Component | hds/form/checkbox/field', function (hooks) {
   test('it should spread all the attributes passed to the component on the input', async function (assert) {
     assert.expect(3);
     await render(
-      hbs`<Hds::Form::Checkbox::Field checked="checked" class="my-class" data-test1 data-test2="test" />`
+      hbs`<Hds::Form::Checkbox::Field checked="checked" class="my-class" name="my-name" data-test />`
```

Comment:
I think this should be `@name` as argument (plus see my other comment on the left)

Reply:
discussed with @alex-ju and he convinced me makes sense this way (we don’t want to encourage the use of props when we can rely on splattributes)